WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -177,6 +179,13 @@ static IEnumerable<(int Size, FeeRate From, FeeRate To)> SplitFeeGroupInBlocks(F
 			.Select(x => x.Size)
 			.Scan(0m, (acc, size) => acc + size);
 
+		var top200Mb = splittedFeeGroups
+			.Zip(
+				accumulatedSizes,
+				(feeGroup, accumulatedSize) => (FeeRate: feeGroup.To, AccumulatedSize: accumulatedSize))
+			.FirstOrDefault(x => x.AccumulatedSize > 200 * Mb)
```

Comment:
it could be, yes.

Reply:
The thing is that this runs also on the client for those using the local node with rpc access. I assumed most users shouldn't touch that because they will not have mempools bigger than 300Mbs (the default value).